zebra-chain/src/commitments/sapling.rs

```diff
@@ -0,0 +1,291 @@
+//! Sapling note and value commitments and types.
+
+#![allow(clippy::unit_arg)]
+
+#[cfg(test)]
+mod arbitrary;
+
+use std::{fmt, io};
+
+use bitvec::prelude::*;
+use rand_core::{CryptoRng, RngCore};
+
+use crate::{
+    keys::sapling::{find_group_hash, Diversifier, TransmissionKey},
+    serde_helpers,
+    serialization::{ReadZcashExt, SerializationError, ZcashDeserialize, ZcashSerialize},
+    types::amount::{Amount, NonNegative},
+};
+
+/// Generates a random scalar from the scalar field 𝔽_{r_𝕁}.
```

Comment:
I'm down with KaTeX, how do we set it up?

Reply:
I created https://github.com/ZcashFoundation/zebra/issues/824 to track this

Reply:
I believe you can still use Unicode characters in KaTeX, i.e. `the scalar field $𝔽\_{r\_𝕁}$.` will render correctly.

Reply:
OK, it sounds like there's consensus on using KaTeX. I made #832 as a quick first go at adding support so that we can get the docs consistent in this PR.  I think it would be better to do a pass converting the math rendering to KaTeX before merging this. I don't have any preference about using unicode in KaTeX, if it works then changing the existing text-math to KaTeX will be less work.